### PR TITLE
! Slider update: 1.2.1 > 1.2.7

### PR DESCRIPTION
**Pull request by XinYenFon**
Signed-off-by: Antes antes@simplemachines.org

### CODE REVIEW

**Comment by live627** [comment]
I'm testing this now and have found that the last two commits are not needed. Furthermore, I would use the below in the template:

```
    <script>
        jQuery("#smf_slider").slippry({
            pause: ', $settings['newsfader_time'],',
            captions: false,
            controls: false,
            useCSS: true
        });
    </script>';
```

**Comment by XinYenFon** [comment]
I was actually going for a cleanup for JS file, I'm aware of those. But that also works. :P

**Comment by XinYenFon** [comment]
@live627 my job is here done. Please test and let me know :)

File: Themes/default/BoardIndex.template.php [hunk 1]
```diff
@@ -35,7 +35,10 @@ function template_newsfader()
 		</ul>
 		<script>
 			jQuery("#smf_slider").slippry({
-				speed: ', $settings['newsfader_time'],'
+				speed: ', $settings['newsfader_time'],',
```
Comment (live627) [inline Themes/default/BoardIndex.template.php]: Speed is the duration during which it fades; pause is the interval between the transitions and seems to be the one we want to stay true to the admin setting..

File: Themes/default/BoardIndex.template.php [hunk 2]
```diff
@@ -35,7 +35,10 @@ function template_newsfader()
...
+				adaptiveHeight: 0,
```
Comment (live627) [inline Themes/default/BoardIndex.template.php:39]: Not a bool.

Comment (XinYenFon) [inline Themes/default/BoardIndex.template.php:39]: I used original values from JS, in this version its using !0 or 0, instead true/false.

**Pull request merged**